Content.Shared/Xenoarchaeology/Artifact/SharedXenoArtifactSystem.Unlock.cs

```diff
@@ -82,11 +82,17 @@ public void FinishUnlockingState(Entity<XenoArtifactUnlockingComponent, XenoArti
             // var activated = ActivateNode((ent, artifactComponent), node.Value, null, null, Transform(ent).Coordinates, false);
             // if (activated)
             soundEffect = unlockingComponent.UnlockActivationSuccessfulSound;
+            // DeltaV - start of node scanner overhaul
+            artifactComponent.LastUnlockingSuccessful = true;
+            // DeltaV - end of node scanner overhaul
```

Comment:
Can just be a single line comment


```suggestion
            artifactComponent.LastUnlockingSuccessful = true; // DeltaV - node scanner overhaul
```